### DISCUSSION TITLE
watchtower: create a new `cmd` directory to enable a standalone watch tower

### DISCUSSION THREAD

**Discussion by Roasbeef** [lightningnetwork/lnd#10215] [Ideas]
As implemented today, a watch tower can only run in a "monolith" mode beside `lnd`. It should be possible to run a binary that _only_ runs a watch tower, to allow for more flexible deployment configurations. Most of the scaffolding is already in place for towers, since they're able to run as a part of `lnd`. The remaining work is to "daemonize" the existing scaffolding such that the tower can run on its own. This stand-lone tower binary (`towerd`?) should come equipped with the current [watch tower sub-server](https://github.com/lightningnetwork/lnd/blob/master/lnrpc/watchtowerrpc/watchtower.proto), and should also be able to interact with using the `lncli tower` command.

**Comment by orbitalturtle** [comment]
Going to take a stab at this.

**Comment by saubyk** [comment]
Converting this to a discussion. Can be reverted to the issue or a new one opened up if any development is planned for this.
